src/Sql/dbo/Stored Procedures/User_BumpAccountRevisionDateByCipherId.sql

```diff
@@ -11,7 +11,7 @@ BEGIN
         U.[AccountRevisionDate] = GETUTCDATE()
     FROM
         [dbo].[User] U
-    LEFT JOIN
+    INNER JOIN
```

Comment:
Is this change of behavior intentional (this sproc is **only** used for users of a specific Organization and `@OrganizationId` will never be `NULL`)?

Reply:
Yes, it is assumed that this sproc is only used for organization owned cipher edits. See `Cipher_Update` sproc for example:

```
    IF @OrganizationId IS NOT NULL
    BEGIN
        EXEC [dbo].[User_BumpAccountRevisionDateByCipherId] @Id, @OrganizationId
    END
    ELSE IF @UserId IS NOT NULL
    BEGIN
        EXEC [dbo].[User_BumpAccountRevisionDate] @UserId
    END
```